lua/telescope/_extensions/terraform_doc/actions.lua

```diff
@@ -41,7 +41,7 @@ function M.url_opener_module(opts)
     local selection = action_state.get_selected_entry()
     actions.close(prompt_bufnr)
     local url = M_api.get_docs_url_module(selection.full_name, selection.provider_name)
-    os.execute(string.format('%s "%s"', opts.url_open_command, url))
+    vim.ui.open(url)
```

Comment:
this is a relative new api that's only available in neovim 0.10.0, so i've made some changes to your mr. 

thanks!